docs/reference/r-releases/r-rev195_149.md

```diff
@@ -0,0 +1,108 @@
+* [DPE-3928] feat: secrets integration (#189)
+* [DPE-5702] chore: Active Controllers alert set to == 0 (#252)
+* [CSS-6503] Add OAuth support for non-charmed external clients (#168)
+* [DPE-5757] Add `extra-listeners` config option (#269)
+
+### Bug fixes
+
+* [DPE-3880] Remove instance field from grafana dashboard (#191) 
+* [DPE-3880] Remove all instances of $job variable in dashboard (#181)
+* [DPE-3900] Remove APT references (#183)
+* [DPE-3932] Fix illegal character on matrix channel (#187)
+* [DPE-4133] Do not change permissions on existing folders when reusing storage (#195)
+* [DPE-4362] fix: alive, restart and alive handling (#202)
+
+### Other changes
+
+* [MISC] Test on juju 3.4 (#190)
+* [MISC] Update package dependencies
+* [DPE-3588] Release documentation update  (#175)
+* [MISC] CI improvements (#209)
+* [DPE-3214] Release 3.6.1 (#179)
+* [DPE-5565] Upgrade dataplatform libs to v38
+* [discourse-gatekeeper] Migrate charm docs (#210, #203, #198, #194, #192)
+* [DPE-3932] Update information in metadata.yaml
+
+## ZooKeeper
+
+New features and bug fixes in the Charmed Apache ZooKeeper operator:
+
+### Features
+
+* [DPE-2285] Refer to Charmhub space from GitHub (#143)
+* [DPE-2597] Re use existing storage (#138)
+* [DPE-3737] Implement ZK client interface (#142)
+* [DPE-3782] Set workload version in install and config hooks (#130)
+* [DPE-3857] Test consistency between workload and metadata versions (#136)
+* [DPE-3869] Secrets in ZK (#129)
+* [DPE-5626] chore: update ZooKeeper up alerting (#166)
+
+### Bug fixes
+
+* [DPE-3880] Remove job variable from dashboard (#134)
+* [DPE-3900] Remove APT references (#131)
+* [DPE-3932] Fix illegal character on matrix channel (#133, #135)
+* [DPE-4183] fix: only handle quorum removal on relation-departed (#146)
+* [DPE-4362] fix: alive, restart and alive handling (#145)
+
+### Other changes
+
+* [DPE-5565] Stable release upgrade
+* chore: bump dp_libs ver (#147)
+* [MISC] General update dependencies (#144)
+* [MISC] Update CI to Juju 3.4 (#137)
+* [DPE-3932] Update information in metadata.yaml
+* [MISC] Update cryptography to 42.0.5
+
+Canonical Data issues are now public on both [Jira](https://warthogs.atlassian.net/jira/software/c/projects/DPE/issues/) 
+and [GitHub](https://github.com/canonical/kafka-operator/issues) platforms.
+
+## Inside the charms
+
+Contents of the Kafka and ZooKeeper charms include:
+
+* Charmed ZooKeeper charm is based on the [charmed-zookeeper snap](https://snapcraft.io/charmed-zookeeper) of the `3/stable` channel (Ubuntu LTS “22.04” - core22-based) that ships the ZooKeeper [3.8.4-ubuntu0](https://launchpad.net/zookeeper-releases/3.x/3.8.4-ubuntu0), built and supported by Canonical
+* Charmed Kafka charm is based on the [charmed-kafka snap](https://snapcraft.io/charmed-kafka) of the `3/stable` channel (Ubuntu LTS “22.04” - core22-based) that ships the Kafka [3.6.1-ubuntu0](https://launchpad.net/kafka-releases/3.x/3.6.1-ubuntu0), built and supported by Canonical
```

Comment:
It's either `Charmed Apache Kafka` or `Apache Kafka charm`.
```suggestion
* Charmed Apache Kafka is based on the [charmed-kafka snap](https://snapcraft.io/charmed-kafka) of the `3/stable` channel (Ubuntu LTS “22.04” - core22-based) that ships the Apache Kafka [3.6.1-ubuntu0](https://launchpad.net/kafka-releases/3.x/3.6.1-ubuntu0), built and supported by Canonical
```

Reply:
Let's go with `Charmed Apache Kafka`